packages/devextreme/js/__internal/ui/m_informer.ts

```diff
@@ -0,0 +1,131 @@
+import registerComponent from '@js/core/component_registrator';
+import type { dxElementWrapper } from '@js/core/renderer';
+import $ from '@js/core/renderer';
+import type { WidgetOptions } from '@js/ui/widget/ui.widget';
+import { getImageContainer } from '@ts/core/utils/m_icon';
+import type { OptionChanged } from '@ts/core/widget/types';
+import Widget from '@ts/core/widget/widget';
+
+const INFORMER_CLASS = 'dx-informer';
+const INFORMER_ALIGNMENT_START_CLASS = 'dx-informer-alignment-start';
+const INFORMER_ALIGNMENT_CENTER_CLASS = 'dx-informer-alignment-center';
+const INFORMER_ALIGNMENT_END_CLASS = 'dx-informer-alignment-end';
+const INFORMER_BG_CLASS = 'dx-informer-bg';
+const INFORMER_TEXT_CLASS = 'dx-informer-text';
+const INFORMER_ICON_CLASS = 'dx-informer-icon';
+
+export interface Properties extends WidgetOptions<Informer> {
+  contentAlignment?: string;
+  icon?: string;
+  showBackground?: boolean;
+  text?: string;
+}
+
+class Informer extends Widget<Properties> {
+  private _$icon?: dxElementWrapper;
+
+  private _$text!: dxElementWrapper;
+
+  _getDefaultOptions(): Properties {
+    return {
+      ...super._getDefaultOptions(),
+      contentAlignment: 'center',
+      icon: '',
+      showBackground: true,
+      text: '',
+    };
+  }
+
+  _initMarkup(): void {
+    const $element = $(this.$element());
+    const { showBackground } = this.option();
+
+    $element.addClass(INFORMER_CLASS);
+    $element.toggleClass(INFORMER_BG_CLASS, showBackground);
+    this._setAlignmentClass();
+
+    super._initMarkup();
+
+    this._renderIcon();
+    this._renderText();
+  }
+
+  _setAlignmentClass(): void {
+    $(this.$element())
+      .removeClass(INFORMER_ALIGNMENT_START_CLASS)
+      .removeClass(INFORMER_ALIGNMENT_CENTER_CLASS)
+      .removeClass(INFORMER_ALIGNMENT_END_CLASS);
+
+    const { contentAlignment } = this.option();
+
+    switch (contentAlignment) {
+      case 'start':
+        $(this.$element()).addClass(INFORMER_ALIGNMENT_START_CLASS);
```

Comment:
Can really `this.$element()` return not a jQuery-like element?

Reply:
yes

Reply:
oh sorry, I confused this with another function. u're right, we can get rid of this wrap